doc/api/tls.md

```diff
@@ -876,6 +876,31 @@ socket.on('end', () => {
 });
 ```
 
+When using an instance of `net.Socket`, use `net.Socket` to upgrade an existing socket. 
```

Comment:
Sorry, this is still 88 chars) But it is a small nit and can be fixed later on landing)